fix_vector_store_rag.py

```
@@ -0,0 +1,45 @@
+            if (
+                node.get("data")
+                and node["data"].get("type") == "AstraDB"
+                and "node" in node["data"]
+                and "template" in node["data"]["node"]
+            ):
+                template = node["data"]["node"]["template"]
+                for key in ["password", "token", "keyspace", "secure_connect_bundle"]:
+                    if key in template:
+                        del template[key]
+
+    with Path(file_path).open("w") as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)